brave/browser-laptop
# Clears old entries from the table

Resolves #14310

## Submitter Checklist:


## Test Plan:
**you need to have a profile stat is older then one month and was not used (with payments turned on and containing some publishers). If you don't have it contact me and I will send you the profile**

(DO THIS ON OTHER BRANCH)
- run old profile (should be older then 1month, payments enabled)
- pin some sites and visit some new sites that are not on the table

(ON THIS BRANCH)
- start profile make sure that timestamp is pushed now + 30 days
- make sure that table is the same as it was on a previous branch
- close browser
- change reconcileStamp in the past
- start browser again
- make sure that only pinned sites and newly visited sites are on the table (if this total is not more then 30min, fuzzing should happen)

## Reviewer Checklist:


Tests
